server/server.go

```
@@ -363,6 +366,23 @@ func (s *Server) SpeedTestHandler(w http.ResponseWriter, req *http.Request) erro
 	return nil
 }
 
+func (s *Server) SnapshotHandler(w http.ResponseWriter, req *http.Request) error {
+	data, err := s.Store().Save()
+	if err != nil {
+		http.Error(w, "Unexpected snapshot failure", http.StatusInternalServerError)
+		return nil
+	}
+	if req.FormValue("disk") == "true" {
+		err = ioutil.WriteFile(path.Join(s.peerServer.RaftServer().Path(), fmt.Sprintf("%d.snap", s.peerServer.RaftServer().CommitIndex())), data, 0600)
+		if err != nil {
+			http.Error(w, "snapshot failed: "+err.Error(), http.StatusInternalServerError)
+		}
+		return nil
+	}
+	w.Write(data)
```

Don't ignore the error here.